tools/ldb_cmd.cc

```diff
@@ -3052,6 +3091,12 @@ void ScanCommand::DoCommand() {
       fprintf(stdout, "%.*s : %.*s\n", static_cast<int>(key_slice.size()),
               key_slice.data(), static_cast<int>(val_slice.size()),
               val_slice.data());
+    } else {
+      std::ostringstream oss;
+      WideColumnsHelper::DumpWideColumns(it->columns(), oss, is_value_hex_);
+      fprintf(stdout, "%.*s : %.*s\n", static_cast<int>(key_slice.size()),
+              key_slice.data(), static_cast<int>(oss.str().length()),
+              oss.str().c_str());
```

Comment:
I think it would be nice to somehow visually signal here that what we're printing is a wide-column structure, so we can easily tell plain KVs and entities apart; what do you think? (The same would apply to the iteration in `DBDumperCommand` above.)